yb-voyager/src/queryparser/unsupportedConstructs.go

```diff
@@ -160,3 +166,147 @@ func containsAdvisoryLocksInNodeList(nodes []*pg_query.Node) bool {
 	}
 	return false
 }
+
+func (qp *QueryParser) containsSystemColumns() bool {
+	if qp.ParseTree == nil {
+		log.Infof("parse tree not available for query-%s", qp.QueryString)
+		return false
+	}
+
+	selectStmtNode, isSelectStmt := qp.ParseTree.Stmts[0].Stmt.Node.(*pg_query.Node_SelectStmt)
+	// Note: currently considering only SELECTs but I/U/D dmls are also possible
```

Comment:
do we have any ticket tracking it?

Reply:
Yes i will be doing that undee Complex Queries ticket
Will add separate tickets for each later of required